src/features/ModelPlan/ModelToOperations/_components/FormModal/index.tsx

```diff
@@ -69,16 +76,18 @@ const MTOModal = ({
           {modalTitle}
         </PageHeading>
 
-        {isRequired && (
-          <p className="margin-y-0 text-base">
-            <Trans
-              i18nKey={t('modal.allFieldsRequired')}
-              components={{
-                s: <span className="text-secondary-dark" />
-              }}
-            />
-          </p>
-        )}
+        {isRequired &&
+          modalType !== 'removeCategory' &&
+          modalType !== 'removeSubcategory' && (
+            <p className="margin-y-0 text-base">
+              <Trans
+                i18nKey={t('modal.allFieldsRequired')}
+                components={{
+                  s: <span className="text-secondary-dark" />
+                }}
+              />
+            </p>
+          )}
```

Comment:
Not in love with having a prop for `isRequired` yet still needed a string literal conditional.  I understand that this modal is being used in the context provider and not explicitly passing down those props, but I'm tempted to request a better solution - as this type of conditional can get out of hand if we add more.  Maybe either remove the `isRequired` prop completely and have some type of mapped config for each of the modal forms?  We should also try to homogenize all the modals to use the context.  Right now we seem to be using the same component in two different ways (context provider and no context provider).  Maybe this can be tech debt but would rather them behave and implemented in a similar, expected fashion.  Let me know your thoughts if you this we should try to tackle this now.  As of now even for me its a bit confusing looking at the modal and it props, then at the context.  State is attempting to be handled in two different ways.